static/js/controllers/reports-content.js

```diff
@@ -45,15 +47,45 @@
         setMessageState(group, 'muted', 'scheduled');
       };
 
+      $scope.toggleExpand = function(selection) {
+        if ($scope.selectMode) {
+          if (selection.report || selection.expanded) {
+            selection.expanded = !selection.expanded;
+          } else {
```

Comment:
when does this happen? Does the selection object not always have `report` and `expanded` fields?